debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -1041,4 +1051,48 @@ protected BinaryLogClient getBinlogClient() {
     public BinlogPosition getCurrentBinlogPosition() {
         return new BinlogPosition(client.getBinlogFilename(), client.getBinlogPosition());
     }
+
+    protected void setBinlogSSLSocketFactory() {
+        if (connectionContext.jdbc() != null) {
+            String acceptedTLSVersion = connectionContext.getSessionVariableForSslVersion();
+            if (acceptedTLSVersion != null) {
```

Comment:
Could you please use `io.debezium.util.Strings.isNullOrEmpty(String)` - better safe than sorry in case the non-value will be returned as an empty string

Reply:
Updated, thanks!